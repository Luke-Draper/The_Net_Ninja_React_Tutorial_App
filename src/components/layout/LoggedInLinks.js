import React from "react";
import { NavLink } from "react-router-dom";

const LoggedInLinks = () => {
	return (
		<ul className="right LoggedInLinks">
			<li>
				<NavLink to="/">New Project</NavLink>
			</li>
			<li>
				<NavLink to="/">Log Out</NavLink>
			</li>
			<li>
				<NavLink to="/" className="btn btn-floating red lighten-1">
					LD
				</NavLink>
			</li>
		</ul>
	);
};

export default LoggedInLinks;
