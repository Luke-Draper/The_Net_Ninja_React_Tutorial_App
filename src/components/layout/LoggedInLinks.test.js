import React from "react";
import LoggedInLinks from "./LoggedInLinks";
import { MemoryRouter } from "react-router-dom"; //Required when component uses a Route or Link
import renderer from "react-test-renderer";

test("LoggedInLinks renders without crashing", () => {
	const component = renderer.create(
		<MemoryRouter initialEntries={["/"]}>
			<LoggedInLinks />
		</MemoryRouter>
	);
	expect(component.toJSON()).toMatchSnapshot();
});
