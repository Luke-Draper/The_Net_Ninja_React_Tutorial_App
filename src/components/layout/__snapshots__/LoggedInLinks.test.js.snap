// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoggedInLinks renders without crashing 1`] = `
<ul
  className="right LoggedInLinks"
>
  <li>
    <a
      aria-current={null}
      href="/create"
      onClick={[Function]}
    >
      New Project
    </a>
  </li>
  <li>
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Log Out
    </a>
  </li>
  <li>
    <a
      aria-current="page"
      className="btn btn-floating red lighten-1 active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      LD
    </a>
  </li>
</ul>
`;
